docs/api/instruments/pendulum/index.rst

```diff
@@ -0,0 +1,12 @@
+.. module:: pymeasure.instruments.pendulum
+
```

Comment:
whoops, the above warning ^ means that you need to add a line to pymeasure/docs/api/instruments/index.rst as this is a new manufacturer [docs](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#adding-documentation)

Reply:
Fixed in 6915d29.